src/NumSharp.Core/RandomSampling/np.random.lognormal.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading.Tasks;
+using NumSharp.Backends;
+using NumSharp.Backends.Unmanaged;
+using NumSharp.Generic;
+using NumSharp.Utilities;
+
+namespace NumSharp
+{
+    public partial class NumPyRandom
+    {
+        /// <summary>
+        /// Draw samples from a log-normal distribution.
+        /// Draw samples from a log-normal distribution with specified mean, standard deviation, and array shape.
+        /// Note that the mean and standard deviation are not the values for the distribution itself, but of the underlying normal distribution it is derived from.
+        /// </summary>
+        /// <param name="mean">Mean value of the underlying normal distribution. Default is 0.</param>
+        /// <param name="sigma">Standard deviation of the underlying normal distribution. Should be greater than zero. Default is 1.</param>
+        /// <param name="dims">Output Shape</param>
+        /// <returns>Drawn samples from the parameterized bernoulli distribution.</returns>
+        public NDArray lognormal (double mean, double sigma, params int[] dims)
```

Comment:
Please add an overload that accepts `Shape shape`, It'll also add support for tuple shapes `(1, 3 ... )`

Reply:
This applies to all methods on `np.*` and `NumPyRandom.*` that you've added..

Reply:
e.g.
```C#
public NDArray lognormal (double mean, double sigma, params int[] dims) => lognormal(mean, sigma, new Shape(dims));
public NDArray lognormal (double mean, double sigma, Shape shape) {
... works with `shape.dimensions` instead of int[] dims
```